app/controllers/sessions_controller.rb

```diff
@@ -1,2 +1,27 @@
 class SessionsController < ApplicationController
+  before_action :require_login, except: [:new, :create]
```

Comment:
Since you apply your before_action globally in your application controller, I think this should be a call to `skip_before_action :require_login, only: [:new, :create]`! :grinning: